box-content-sdk/src/main/java/com/box/androidsdk/content/requests/BoxRequestsFile.java

```diff
@@ -1159,4 +1193,52 @@ protected void onSendCompleted(BoxResponse<BoxVoid> response) throws BoxExceptio
             super.handleUpdateCache(response);
         }
     }
+
+    public static class DownloadRepresentation extends BoxRequestDownload<BoxDownload, DownloadRepresentation> {
+
+        /**
+         * String sufix to be used when fetching a representation page
+         */
+        private static final String REPRESENTATION_PAGE_ASSET_EXTENSION = ".png";
+        protected BoxRepresentation mRepresentation;
+
+        /**
+         * The page number to retrieve in case the representation is paged.
+         * If page is required and not provided, retrieve page 1 as default
+         */
+        protected int mRequestPage = 1;
+
+        public DownloadRepresentation(String id, final File target, BoxRepresentation representation, BoxSession session) {
+            super(id, BoxDownload.class, target, representation.getContent().getUrl(), session);
+            mRepresentation = representation;
+        }
+
+        @Override
+        protected URL buildUrl() throws MalformedURLException, UnsupportedEncodingException {
+            String assetPathReplacement = "";
+            if(isPaged()) {
+                assetPathReplacement = String.format(Locale.ENGLISH, "%d%s", mRequestPage, REPRESENTATION_PAGE_ASSET_EXTENSION);
+            }
+            return new URL(mRequestUrlString.replace(BoxRepresentation.BoxRepContent.ASSET_PATH_STRING,
+                                                     assetPathReplacement));
+        }
+
+        /**
+         * Returns whether this request supports specifying a page
+         * @return whether this representation is paged
+         */
+        public boolean isPaged() {
+            JsonValue jsonPaged = mRepresentation.getProperties().getPropertyValue(BoxRepresentation.FIELD_PROPERTIES_PAGED);
```

Comment:
Can we make a class like RepresentationProperties that extends BoxMap? Then maybe within that object we can have a helper method isPaged.